package.json

```diff
@@ -5,14 +5,16 @@
   "main": "lib/index.js",
   "typings": "./index.d.ts",
   "scripts": {
+    "qa": "run-p lint test",
     "lint-src": "./node_modules/.bin/eslint src/**/*.js",
     "lint-test": "./node_modules/.bin/eslint test/**/*.js",
-    "lint": "npm run lint-src && npm run lint-test",
+    "lint": "run-p lint-*",
```

Comment:
Can you help me understand a bit more of what is going on here?

Reply:
This is using `npm-run-all` to execute task in parallel. Before they were running sequentially (`&&`)

It's cross platform and supports globs, like in this case, where it will execute in parallel all scripts that start with `lint-`.